firmware/shared/controls/DI_driverTorqueRequest.h

```diff
@@ -1,25 +1,14 @@
 #pragma once
 #include <cmath>
-#include <iostream>
-
 namespace ctrl {
 
 // Templating means this function will work on any types that support addition.
-template <typename T, typename B>
-T DI_driverTorqueRequest(T READY_TO_DRIVE, T DI_Error, B AccelPedPos) {
-    T u1 = 0.0;
-    if ((!READY_TO_DRIVE) == 1 || DI_Error == 1) {
-        return u1;
+template <typename B>
```

Comment:
your templates look good, only thing is to rename `B` to the more standard template variable `T`